packages/suite/src/components/firmware/CheckSeed.tsx

```diff
@@ -12,61 +12,45 @@ const CheckboxRow = styled.div`
     justify-content: center;
 `;
 
-const CheckSeedStep = () => {
+type Props = {
+    onSuccess: () => void;
+};
+
+const CheckSeedStep = ({ onSuccess }: Props) => {
     const { device } = useDevice();
-    const { toggleHasSeed, hasSeed, setStatus } = useFirmware();
+    const { toggleHasSeed, hasSeed } = useFirmware();
 
     // unacquired device handled on higher level
     if (!device?.features) return null;
 
     // device is not backed up - it is not advisable to do firmware update
-    if (device.features.needs_backup || device.features.unfinished_backup) {
-        return (
-            <OnboardingStepBox
-                image="FIRMWARE"
-                heading={
-                    <Translation
-                        id="TR_DEVICE_LABEL_IS_NOT_BACKED_UP"
-                        values={{ deviceLabel: device.label }}
-                    />
-                }
-                description={<Translation id="TR_FIRMWARE_IS_POTENTIALLY_RISKY" />}
-                outerActions={
-                    <Button
-                        onClick={() => setStatus('waiting-for-bootloader')}
-                        data-test="@firmware/confirm-seed-button"
-                        isDisabled={!device?.connected || !hasSeed}
-                    >
-                        <Translation id="TR_CONTINUE" />
-                    </Button>
-                }
-                disableConfirmWrapper
-                nested
-            >
-                <CheckboxRow>
-                    <Checkbox
-                        isChecked={hasSeed}
-                        onClick={toggleHasSeed}
-                        data-test="@firmware/confirm-seed-checkbox"
-                    >
-                        <P>
-                            <Translation id="FIRMWARE_USER_TAKES_RESPONSIBILITY_CHECKBOX_DESC" />
-                        </P>
-                    </Checkbox>
-                </CheckboxRow>
-            </OnboardingStepBox>
-        );
-    }
-
     // expected flow - device is backed up
+    const notBackedUp = device.features.needs_backup || device.features.unfinished_backup;
```

Comment:
nit:
```suggestion
    const isBackedUp = !device.features.needs_backup && !device.features.unfinished_backup;
```

I would make it positive

Reply:
Fixed in [`f6db9b4a`](https://github.com/trezor/trezor-suite/commit/f6db9b4a0c1c272e8e09eee978cdc3e648e8c443).